### PR TITLE
Adding the auth user tests

### PR DESCRIPTION
**Pull request by Achilleasein**
#### Description:

- _Adding the require_emergency_target_auth and require_singleuser_auth tests to the CIS profile for rhel7._

### CODE REVIEW

**Comment by Achilleasein** [comment]
Moved the changes as requested by Matus and adding again the comments from the last PR.

- Please move your changes in your fork from master into another branch.
- I would also change the name of the test scripts, they don't need to contain rule ID, file names are missing pass/fail at the end.
- Remove cis from the file name as they are applicable also to other profiles. So change it to something like rhel7_correct_value.pass.sh.

**Comment by matusmarhefka** [comment]
@Achilleasein Make the `wrong_value.fail.sh` scenarios of both rules applicable on all RHELs.

File: linux_os/guide/system/accounts/accounts-physical/require_singleuser_auth/tests/rhel7_correct_value.pass.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+{{% if init_system == "systemd" -%}}
+service_file="/usr/lib/systemd/system/rescue.service"
+sulogin='/bin/sh -c "/usr/sbin/sulogin; /usr/bin/systemctl --fail --no-block default"'
+if grep "^ExecStart=.*" "$service_file" ; then
+    sed -i "s%^ExecStart=.*%ExecStart=-$sulogin%" "$service_file"
+else
+    echo "ExecStart=-$sulogin" >> "$service_file"
+fi
+{{%- else -%}}
```
Comment (matusmarhefka) [inline linux_os/guide/system/accounts/accounts-physical/require_singleuser_auth/tests/rhel7_correct_value.pass.sh:13]: This is test scenario for RHEL7 so pelase remove the whole part in the `else`, it is not needed.

Comment (Achilleasein) [reply linux_os/guide/system/accounts/accounts-physical/require_singleuser_auth/tests/rhel7_correct_value.pass.sh:10]: done

File: linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/tests/rhel7_correct_value.pass.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# platform = multi_platform_rhel, multi_platform_fedora
```
Comment (matusmarhefka) [inline linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/tests/rhel7_correct_value.pass.sh:3]: Restore to RHEL7 and add `multi_platform_rhel` to fail scenarios.

**Review by matusmarhefka — APPROVED** [review]

**Pull request merged**